CleverFerret/src/main/java/com/universalmedialibrary/services/tts/GoogleCloudTtsService.kt

```
@@ -0,0 +1,277 @@
+
+            (0 until voicesArray.length()).map { i ->
+                val voice = voicesArray.getJSONObject(i)
+                val languageCodes = voice.getJSONArray("languageCodes")
+                CloudVoiceInfo(
+                    name = voice.getString("name"),
+                    languageCode = if (languageCodes.length() > 0) languageCodes.getString(0) else "",
+                    gender = voice.optString("ssmlGender", ""),
+                    type = if (voice.getString("name").contains("Neural")) "Neural" else "Standard"
+                )
+            }
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)